srcpkgs/rm-improved/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'rm-improved'
+pkgname=rm-improved
+version=0.13.1
+revision=1
+build_style=cargo
+short_desc="Safe and ergonomic alternative to rm"
+maintainer="Juan <juan@horlux.org>"
+license="GPL-3.0-only"
+homepage="https://github.com/nivekuil/rip"
+distfiles="https://crates.io/api/v1/crates/rm-improved/${version}/download>rm-improved-${version}.tar.gz"
+checksum=6964bbdb1633b3af61aa3488175d111bb5cc84e56cd668255e0fb7dce4008722
+
+if [ "$CROSS_BUILD" ]; then
+	export HOST_CC="$BUILD_CC"
+	export HOST_CFLAGS="$BUILD_CFLAGS"
+	export BINDGEN_EXTRA_CLANG_ARGS="--sysroot=$XBPS_CROSS_BASE"
+fi
```

Comment:
Any reason this is needed specifically for this package but not for others? Could make sense to add it to the build_style.